mitxgraders/helpers/compatibility.py

```diff
@@ -29,31 +29,14 @@ def _wrapper(f):
         return _f
     return _wrapper
 
-# This is copied from six repo; was added in version 1.12.0, but edX uses 1.11.0
-def ensure_text(s, encoding='utf-8', errors='strict'):
-    """Coerce *s* to six.text_type.
-    For Python 2:
-      - `unicode` -> `unicode`
-      - `str` -> `unicode`
-    For Python 3:
-      - `str` -> `str`
-      - `bytes` -> decoded to `str`
-    """
-    if isinstance(s, six.binary_type):
```

Comment:
I'm worried about this. We sometimes dealt with bytes objects, which need to be decoded. You can't just call `str` on that.

Reply:
Or maybe not, if this was literally the only place we used ensure_text... hmm. *shrug*. If the tests are happy, it's probably fine - we had perfect test coverage.